sklearn/calibration.py

```diff
@@ -207,6 +194,22 @@ def fit(self, X, y, sample_weight=None):
             calibrated_classifier.fit(X, y, sample_weight)
             self.calibrated_classifiers_.append(calibrated_classifier)
         else:
+            X, y = self._validate_data(
+                X, y, accept_sparse=['csc', 'csr', 'coo'],
+                force_all_finite=False, allow_nd=True
+            )
+
+            # Check that each cross-validation fold can have at least one
+            # example per class
+            n_folds = self.cv if isinstance(self.cv, int) \
+                else self.cv.n_folds if hasattr(self.cv, "n_folds") else None
+            if n_folds and \
```

Comment:
Here you can use external `( )` instead of `\`

Reply:
Not sure of my formatting here so let me know if not idea.